src/winter_onboarding_2021/fleet_management_service/core.clj

```diff
@@ -1,13 +1,13 @@
 (ns winter-onboarding-2021.fleet-management-service.core
   (:require [ring.adapter.jetty :as raj]
-            [ring.logger :as logger]
             [bidi.ring :as br]
             [mount.core :as mount :refer [defstate]]
             [ring.middleware.multipart-params :refer [wrap-multipart-params]]
             [ring.middleware.flash :refer [wrap-flash]]
             [ring.middleware.params :refer [wrap-params]]
             [ring.middleware.keyword-params :refer [wrap-keyword-params]]
             [ring.middleware.session :refer [wrap-session]]
+            [winter-onboarding-2021.fleet-management-service.logger]
```

Comment:
Is this an unused namespace?

Reply:
@neenaoffline & I were experimenting the `mount` library's start order. Will remove this unused namespace.